docs/package.json

```diff
@@ -15,7 +15,9 @@
   "dependencies": {
     "@vuepress/plugin-html-redirect": "^0.1.4",
     "@vuepress/plugin-medium-zoom": "^1.8.2",
+    "markdown-it": "^13.0.1",
```

Comment:
See comment at the top of the PR. Only required to support MultiMarkdown tables if we need some.